sdk/core/azure-core/CMakeLists.txt

```diff
@@ -10,12 +10,12 @@ set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../cmake-modules")
 
-include(az_vcpkg)
-include(az_version)
-include(CodeCoverage)
-include(DefineTransportAdapter)
-include(doxygen_common)
-include(global_compile_options)
+include(AzureVcpkg)
```

Comment:
Missed the newly added `download_test_data.cmake` - PascalCase it. Also, let's add `Json` in the name.

@RickWinter do we want to prefix cmake modules that came from external sources with `Azure` as well (so `AzureDownloadJsonTestData`?